doc/tutorial/examples.txt

```diff
@@ -285,6 +282,54 @@ expression that evaluates to a tensor of same shape and dtype.
     different pattern, just pass it as a parameter
     ``theano.shared(..., broadcastable=(True, False))``
 
+Copying functions
+=================
+Theano functions can be copied, which can be useful for creating similar
+functions but with different shared variables or updates. This is done using
+the :func:`copy()<theano.compile.function_module.Function.copy>` method of ``function`` objects. The optimized graph of the original function is copied,
+so compilation only needs to be performed once.
+
+Let's start from the accumulator defined above:
+
+>>> import theano
+>>> import theano.tensor as T
+>>> state = theano.shared(0)
+>>> inc = T.iscalar('inc')
+>>> accumulator = theano.function([inc], state, updates=[(state, state+inc)])
+
+We can use it to increment the state as usual:
+
+>>> accumulator(10)
```

Comment:
So, a plugin for the documentation is treating those inline examples as unit tests to be checked.
In fact, it is running these lines, and comparing the result with the expected result that it tries to find on the next.
In that case, `accumulator(10)` returns `array(0)`, but since there is nothing on the next line, it is treated as a failure (it figures that call should return `None`).

I think this is the same for the next failures as well.